src/man/nvim.1

```diff
@@ -387,10 +387,10 @@ features like
 .El
 .Sh FILES
 .Bl -tag -width "~/.config/nvim/init.vim"
```

Comment:
Does this line need change?

Reply:
It does not need to change as it is the same length.  As in "vim" is the same length/width as "lua"

https://linux.die.net/man/7/groff_mdoc  Documents the `-width` parameter.

But it would not hurt to change it and may make it less confusing to change it.